### PR TITLE
Make canarypush.py to handle two canarypush jobs

### PR DESCRIPTION
**Pull request by krzyzacy**
local run gives
```
W0303 17:55:39.469] Error response from daemon: Get https://registry-1.docker.io/v2/: unauthorized: incorrect username or password
```
which is expected to work on Jenkins

### CODE REVIEW

**Comment by krzyzacy** [comment]
cc @timstclair @piosz as the owner of the jobs, FYI

**Comment by timstclair** [comment]
lgtm

File: jenkins/job-configs/kubernetes-jenkins/bootstrap-ci-dockerpush.yaml [hunk 1]
```diff
@@ -6,9 +6,11 @@
         './test-infra/jenkins/bootstrap.py' \
             --branch='{branch}' \
             --job='{job-name}' \
+            --json \
```
Comment (fejta) [inline jenkins/job-configs/kubernetes-jenkins/bootstrap-ci-dockerpush.yaml]: AWESOME!!!

**Comment by fejta** [comment]
/lgtm
FYI I resolved the config.json conflicts for you

**Comment by krzyzacy** [comment]
@fejta thanks :-)

**Pull request merged**